packages/@cdklabs/cdk-cicd-wrapper/src/stacks/VPCStack.ts

```diff
@@ -120,19 +161,20 @@ export class VPCStack extends cdk.Stack {
   }
 
   /**
-   * Launches a VPC with a private subnet with egress and a public subnet.
+   * Launches a VPC with a private subnet with egress.
    * @param props The properties for configuring the VPC.
    * @returns The created VPC.
    */
   private launchVPCWithEgress(props: VPCStackProps) {
+    this.subnetType = ec2.SubnetType.PRIVATE_WITH_EGRESS;
```

Comment:
As I see at the moment it is not configurable and we have the same line in 2 places. Can't we just use a constant then?

Reply:
Its not the same line, its depending on the methods, for launchVPCIsolated the subnetType is PRIVATE_ISOLATED and for the launchVPCWithEgress the subnetType is PRIVATE_WITH_EGRESS.